utils/clflags.ml

```diff
@@ -358,3 +358,18 @@ let parse_color_setting = function
 let color = ref Misc.Color.Auto ;; (* -color *)
 
 let unboxed_types = ref false
+
+let arg_spec = ref []
+let arg_names = ref Misc.StringMap.empty
+let add_arguments loc args =
+  List.iter (function (arg_name, _, _) as arg ->
+    try
+      let loc2 = Misc.StringMap.find arg_name !arg_names in
+      Printf.eprintf
+        "Warning: plugin argument %s is already defined:\n" arg_name;
```

Comment:
The warning is not in emacs format, so it will get lost in the noise of the build. I'm not sure what can be done about this.